src/layer.rs

```diff
@@ -350,7 +359,7 @@ impl Layer {
     }
 
     /// Returns a reference the glyph with the given name, if it exists.
-    pub fn get_glyph<K>(&self, glyph: &K) -> Option<&Arc<Glyph>>
+    pub fn glyph<K>(&self, glyph: &K) -> Option<&Arc<Glyph>>
```

Comment:
Wouldn't `get` be more appropriate? Layers are basically HashMaps with fixed types...

Reply:
I think there are enough other things hanging of a layer that just 'get' would be unclear; it could be the layer's name, or it's path, or something from the lib...

Reply:
Then I actually advocate keeping "get_glyph" because it is clear. Otherwise, I'll bite my tongue now.